### PR TITLE
Add simple regression for SynthFeatureLargeConfigSpace

### PR DESCRIPTION
**Pull request by boehmseb**

### CODE REVIEW

**Comment by boehmseb** [comment]
Start revision has to be updated once se-sic/FeaturePerfCSCollection#22 is merged.

File: SynthFeatureLargeConfigSpace/regression_1.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/projects/SynthFeatureLargeConfigSpace/main.cpp b/projects/SynthFeatureLargeConfigSpace/main.cpp
+index fbf505d..766484f 100644
+--- a/projects/SynthFeatureLargeConfigSpace/main.cpp
++++ b/projects/SynthFeatureLargeConfigSpace/main.cpp
+@@ -51,7 +51,7 @@ int main(int argc, char *argv[]) {
+ 
+   if (F5) {
+     if (F7 && !F3) {
+-      sleepTime *= 3;
++      sleepTime *= 4;
```
Comment (vulder) [inline SynthFeatureLargeConfigSpace/regression_1.patch:10]: Maybe a larger regression would be useful, only 1 millisecond might be hard to measure

Comment (vulder) [reply SynthFeatureLargeConfigSpace/regression_1.patch:10]: ok, the it should be ok

File: SynthFeatureLargeConfigSpace/regression_1.info [hunk 2]
```diff
@@ -0,0 +1,10 @@
+description: "Inserts regression into the sleep time computation"
+include_revisions:
+  revision_range:
+    start: a0b2c1d6fa1de2c50fda93b6ca508b605a5f6f95
+    end:   master
+path: regression_1.patch
+project_name: FeaturePerfCSCollection
+shortname: reg_long_sleep
+tags:
+- regression
```
Comment (vulder) [inline SynthFeatureLargeConfigSpace/regression_1.info:10]:
```suggestion
- regression
- perf_prec
```

**Review by vulder — APPROVED** [review]
Otherwise, after updating the commit this looks fine.

File: SynthFeatureLargeConfigSpace/regression_1.info [hunk 3]
```diff
@@ -0,0 +1,10 @@
+description: "Inserts regression into the sleep time computation"
+include_revisions:
+  revision_range:
+    start: a0b2c1d6fa1de2c50fda93b6ca508b605a5f6f95
```
Comment (boehmseb) [inline SynthFeatureLargeConfigSpace/regression_1.info:4]:
```suggestion
    start: 6863c78c242a8e67b09d2947bd83ef09ceff4be4
```

**Pull request merged**